app/views/preferences_anything_else/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<% content_for :header_title, "What else should we know?" %>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="step-section-header">
+      Is there anything else you'd like us to know about your situation?
+
+      <div class="step-section-header__subhead">
+        Feel free to give other details you think might be helpful for us to know. Otherwise, you can skip this question.
+      </div>
+    </div>
+  </header>
+
+  <div class="form-card__content">
+    <%= form_for @step, as: :step, builder: MbFormBuilder, url: current_path, method: :put do |f| %>
+      <%= f.mb_textarea :anything_else, "Anything else?", options: { placeholder: "Write here" } %>
```

Comment:
I think there is an ongoing preference to not use placeholder text  https://trello.com/c/CstrMJ8l/85-design-as-a-client-i-want-the-most-usable-form-fields-possible-which-means-no-placeholder-text